extension/popup/set_view_mod.js

```diff
@@ -1,66 +1,75 @@
 document.console.log('ss');
-function listenForClicks() {
-    function setChecks(tabs) {
-        let msg =[];
-        if (document.getElementById('text_white').checked){
-            msg.push("tw");
-        }
-        if (document.getElementById('text_crossed').checked){
-            msg.push("tc");
-        }
-        if (document.getElementById('text_removed').checked){
-            msg.push("tr");
-        }
-        msg = msg.join();
-        function onError(error) {
-            alert(`Error: ${error}`);
-        }
-        browser.storage.sync.set({style: msg});
-        browser.tabs.sendMessage(tabs[0].id, {
-            command: msg,
-        });
-    }
-
-    browser.tabs.query({active: true, currentWindow: true})
-        .then(setChecks)
-        .catch(reportError);
-
-    function reportError(error) {
-        console.error(`Could not opt-out: ${error}`);
-        // reportExecuteScriptError(error)
-    }
-}
-
 /**
  * There was an error executing the script.
  * Display the popup's error message, and hide the normal UI.
  */
 function reportExecuteScriptError(error) {
```

Comment:
Moved this up and activated the call to it again, maybe it works now that it is called after it is defined?